pkg/ddevapp/ddevapp_test.go

```diff
@@ -235,6 +235,20 @@ var (
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "This is a test page"},
 			FilesImageURI:                 "/sites/default/files/2022-07/Logo.png",
 		},
+		// 13: craftcms
+		{
+			Name:                          "TestPkgCraftCms",
+			SourceURL:                     "https://github.com/khalwat/ddev/files/9497111/cms-4.2.3.zip",
```

Comment:
We have to remember to move these over to the official junk testing release. Can do now.

Reply:
Go ahead and update the URLs to where you have moved them to if you like.

Reply:
This is complete.